addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/report/ExtensionAuthhelperReport.java

```diff
@@ -174,22 +179,41 @@ public void handle(ReportData reportData) {
                          * The AUTH_SUCCESS_STATS / AUTH_FAILURE_STATS stats can get raised on another domain.
                          * Any successes are good, but just failures are bad.
                          */
-                        addSummaryItem(
-                                ard,
-                                "auth",
+                        boolean hasSuccessStats =
+                                inMemoryStats.getStat(
+                                                hostname, AuthenticationHelper.AUTH_SUCCESS_STATS)
+                                        != null;
+                        boolean hasFailureStats =
+                                inMemoryStats.getStat(
+                                                hostname, AuthenticationHelper.AUTH_FAILURE_STATS)
+                                        != null;
+                        boolean overAllStatus =
```

Comment:
Why overAll?

Reply:
What would you rather it be?

To me it's the overall authentication status or result.

Reply:
`overAll` vs `overall`

Reply:
Oh okay, sorry I misunderstood. Will tweak.